### DISCUSSION TITLE
Problem with export ending

### DISCUSSION THREAD

**Discussion by djp4nd4** [Zealot111/OCAP#12] [Q&A]
**OCAP Version:** `last`
**Server OS:** `Windows`

**Description:**
I have a problem with export ending. Until september, all works fine, but now after export i see a lot of error message in log file.

**Steps to reproduce:**

**RPT File**
[ocap-ext.20201001_192755.log](https://github.com/Zealot111/OCAP/files/5313890/ocap-ext.20201001_192755.log)
[ocap-main.20201001_192755.log](https://github.com/Zealot111/OCAP/files/5313891/ocap-main.20201001_192755.log)

**Comment by Zealot111** [comment]
@djp4nd4 it seems that your framework call endMission function in extension with incorrect parameters
From log `:EVENT: 3:[402::"endMission"::[string,<null>]] `
this is incorrect parameters instead 'string' you should have some actual string, for example ["EAST","OPFOR Wins. Their enemies suffered heavy losses!"]
What parameters do you provide when call [] call ocap_fnc_exportData; ?

**Comment by djp4nd4** [comment]
for now i'm not provide any parameters. This is my code
                            if ((["InfSide0"] call FHQ_TT_areTasksSuccessful)) then {
				if (isServer) then {
					[] call ocap_fnc_exportData;
				};
			   ["end1", true, 7] remoteExec ["BIS_fnc_endMission"];
                           ;

**Comment by tekig** [comment]
You need to call, for example, like this
```sqf
[east, "OPFOR Wins. Their enemies suffered heavy losses!"] call ocap_fnc_exportData;
```
or
```sqf
["OPFOR Wins. Their enemies suffered heavy losses!"] call ocap_fnc_exportData;
```

You can also try creating an event at the end of the game on the server side. We've had problems with this in the past. BIS could fix this. (Issue #3) and [BIS Tracker](https://feedback.bistudio.com/T120253)
```sqf
addMissionEventHandler ["MPEnded", { ["End mission"] call ocap_fnc_exportData; }];
```

**Comment by djp4nd4** [comment]
In the initServer.sqf I have inserterd this row
addMissionEventHandler ["MPEnded", {[west,"Missione conclusa!"] call ocap_fnc_exportData}];
The capture stop well but i have an error in log.
Append Log
[ocap-ext.20201008_000242.log](https://github.com/Zealot111/OCAP/files/5344125/ocap-ext.20201008_000242.log)
[ocap-main.20201008_000242.log](https://github.com/Zealot111/OCAP/files/5344129/ocap-main.20201008_000242.log)

**Comment by tekig** [comment]
You haven't set any markers or are not using SWT markers? This is not critical if you have no problems displaying the mission on the site.

**Comment by Zealot111** [comment]
@djp4nd4 this error is not a problem and means that you have no recorded markers. You need use specially modified version (by us) of swt_markers mod to record side channel markers.
